### DISCUSSION TITLE
Icon Request: fa-skate

### DISCUSSION THREAD

**Discussion by lizabbey** [FortAwesome/Font-Awesome#21206] [Icon Requests]
Used to promote roller derby, roller rinks or roller skating in general. 

![](http://www.wordans.ca/wvc-1258088400/wordansfiles/images/2009/8/28/22072/22072_340.jpg)

![](https://s-media-cache-ak0.pinimg.com/236x/19/87/21/19872141d65fb991a3c288e4c4ebc389.jpg)

**Comment by oriadam** [comment]
+1 
Also Rollerblading  rollerblades  rollerskate and ice skating
![](https://cdn4.iconfinder.com/data/icons/hotel-and-restaurant-vol-1-1/48/51-128.png)
http://www.iconarchive.com/download/i90911/icons8/windows-8/Holidays-Christmas-Ice-Skate.ico

**Comment by ipv6freely** [comment]
+1 adding a hockey skate would be great too. Something like:
<img width="177" alt="screen shot 2018-02-09 at 11 30 34 am" src="https://user-images.githubusercontent.com/16598321/36046100-b2a68a48-0d8c-11e8-9d85-1e764c84e584.png">

**Comment by CalumChilds** [comment]
+1

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🛼 [U+1F6FC](https://emojipedia.org/emoji/%F0%9F%9B%BC/) Roller Skate

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Reply by ipv6freely** [reply]
Awesome, thank you! Hopefully you consider the hockey skate too!
